server.js

```diff
@@ -4246,11 +4246,18 @@ cache(function(data, match, sendBadge, request) {
     try {
       var data = JSON.parse(buffer);
       var issues = data.count;
-      badgeData.text[1] = issues + (isRaw? '': ' open');
+      if (res.statusCode !== 200) {
```

Comment:
It's already merged, but I think next time we could check status code before parsing response body.